Public/Src/Cache/ContentStore/Distributed/NuCache/TransitioningContentLocationStore.cs

```diff
@@ -27,23 +28,28 @@ namespace BuildXL.Cache.ContentStore.Distributed.NuCache
     /// <summary>
     /// <see cref="IContentLocationStore"/> implementation that supports old redis and new local location store.
     /// </summary>
-    internal class TransitioningContentLocationStore : StartupShutdownBase, IContentLocationStore, IDistributedLocationStore
+    internal class
+        TransitioningContentLocationStore : StartupShutdownBase, IContentLocationStore, IDistributedLocationStore

```

Comment:
nit: can you please undo this formatting?

Reply:
fixed